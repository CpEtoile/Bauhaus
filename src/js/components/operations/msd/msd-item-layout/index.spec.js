import React from 'react';
import renderer from 'react-test-renderer';
import MSDItemLayout from './';

describe('MSDItemLayout', () => {
	it('should return the right template', () => {
		const tree = renderer
			.create(<MSDItemLayout title={'title'}>CHILDREN</MSDItemLayout>)
			.toJSON();
		expect(tree).toMatchSnapshot();
	});
});
