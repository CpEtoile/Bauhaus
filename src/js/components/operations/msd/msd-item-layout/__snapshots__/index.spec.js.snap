// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MSDItemLayout should return the right template 1`] = `
<article
  className="panel panel-default contenu"
>
  <div
    className="panel-heading"
  >
    <h3>
      title
    </h3>
  </div>
  <div
    className="panel-body"
  >
    CHILDREN
  </div>
</article>
`;
